dev/tests/verification/TestModule/Test/BasicFunctionalTest.xml

```diff
@@ -76,6 +76,7 @@
         <grabTextFrom selector=".functionalTestSelector" stepKey="grabTextFromKey1" />
         <grabValueFrom selector=".functionalTestSelector" stepKey="grabValueFromKey1" />
         <magentoCLI command="maintenance:enable" arguments="&quot;stuffHere&quot;" stepKey="magentoCli1"/>
+        <magentoCLI command="maintenance:enable" arguments="&quot;stuffHere&quot;" timeout="120" stepKey="magentoCli2"/>
```

Comment:
Can we add two `<magentoCLISecret ..>` actions to this verification test? Use default and 120 timeout like magentoCLI.

Reply:
Added the steps to this verification test.